SWUpdate warnings

```
$ swupdate -i iot2050-image-swu-example-iot2050-debian-iot2050.swu
Swupdate v2021.11.0

Licensed under GPLv2. See source distribution for detailed copyright notices.

[INFO ] : SWUPDATE started :  Software Update started !
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd0 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd1 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd2 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd3 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd4 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd6 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd7 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd8 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd9 - maybe not a NAND or raw device
[ERROR] : SWUPDATE failed [0] ERROR : cannot attach mtd10 - maybe not a NAND or raw device
```

@gylstorffq, can we get rid of these error messages and the underlying probing of mtdX?

--- a/recipes-core/swupdate/swupdate_%.bbappend
+++ b/recipes-core/swupdate/swupdate_%.bbappend
@@ -7,8 +7,5 @@
 # SPDX-License-Identifier: MIT
 
 # Set pacakge build options
-# cross: for crosscompiling
-# nodocs: for no documentation
-# nocheck: do not build test
 # pkg.swupdate.bpo: Avoid mtd read errors during swupdate processing
-SWUPDATE_BUILD_PROFILES += "cross nodoc nocheck pkg.swupdate.bpo"
+DEB_BUILD_PROFILES += "pkg.swupdate.bpo"

